test/src/comps/test_util.cc

```diff
@@ -0,0 +1,26 @@
+#include <gtest/gtest.h>
+#include <blas.hh>
+#include <RandBLAS.hh>
+#include <RandLAPACK.hh>
+#include <math.h>
+
+#include <chrono>
+using namespace std::chrono;
+
+#define RELDTOL 1e-10;
+#define ABSDTOL 1e-12;
+
+class TestUtil : public ::testing::Test
+{
+    protected:
+
+    virtual void SetUp() {};
+
+    virtual void TearDown() {};
+
+    template <typename T>
+    static void 
+    test_1()
+    {
+    }
+};
```

Comment:
Three options:

1. Add tests.
2. Write ``TODO: add tests``.
3. Delete this file.